pom.xml

```diff
@@ -97,6 +97,7 @@
         <io.prometheus.simpleclient>0.16.0</io.prometheus.simpleclient>
         <jakarta.validation-api.version>2.0.2</jakarta.validation-api.version>
         <swagger-annotations.version>2.2.11</swagger-annotations.version>
+        <org.springframework.web.version>5.3.33</org.springframework.web.version>
```

Comment:
Why not 5.3.34? Isn't that the latest?

Reply:
Yes, I chose 5.3.33 because it's the patched version for the vulnerability, but I can update to the latest version.